### PR TITLE
Add pouchdb-adapter-hyperbee

### PR DESCRIPTION
**Pull request by RangerMauve**
This acts kinda like a leveldown storage adapter, except any indexes you create can be sparsely queried by remote peers without having to do a full replication pass.

### CODE REVIEW

File: docs/external.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Encrypt a PouchDB/CouchDB database.
 
 Replicate from PouchDB to Dat.
 
+#### [PouchDB Hyperbee](https://github.com/RangerMauve/pouchdb-adapter-hyperbee)
+
+Sparesely load data from Hyperbee over p2p networks in Node and the Browser.
```
Comment (daleharvey) [inline docs/external.md:37]: Sorry little type here? "Sparsely"

**Review by daleharvey — CHANGES_REQUESTED** [review]
Looks like a great project, thanks for the add, happy to merge once we fix up the typo

File: docs/external.md (hunk 1 above)
Comment (RangerMauve) [reply docs/external.md]: Thanks! Fixed.

**Review by daleharvey — APPROVED** [review]
Great cheers

**Comment by RangerMauve** [comment]
Thank you 😁

**Pull request merged**